pkg/gen/nodejs-templates/yaml.ts.mustache

```diff
@@ -87,7 +87,7 @@ import * as outputs from "../types/output";
 
     export interface ConfigOpts {
         /** JavaScript objects representing Kubernetes resources. */
-        objs: pulumi.Input<pulumi.Input<any>[]>;
```

Comment:
Has this always been an inputty type that just hasn't worked?

Reply:
Nope, this value was previously prompt (only usage occurred inside resolved Outputs/Promises).